src/api/app/controllers/trigger_controller.rb

```diff
@@ -28,7 +28,12 @@ def rebuild
   def release
     raise NoPermissionForPackage.setup('no_permission', 403, "no permission for package #{@pkg} in project #{@pkg.project}") unless policy(@pkg).update?
 
-    matched_repo = @pkg.project.repositories.includes(:release_targets).any? { |repo| matched_repo?(repo) }
+    matched_repo = FALSE
+    @pkg.project.repositories.includes(:release_targets).each do |repo|
+      if matched_repo?(repo) then
+        matched_repo = TRUE
```

Comment:
do not use FALSE/TRUE constants, they are deprecated. You can just use `false` and `true`.

Reply:
ok, updating...